.github/issue_template.md

```diff
@@ -64,7 +64,9 @@
   - Post a screenshot or the command the triggered the problem, if applicable.
   - Avoid using ambiguous phrases like: doesn't work, there'a problem, etc.
     Help us reproduce the issue by explaining what went wrong, and what did you expect to happen.
-  - Read the following page to avoid posting a bad issue: https://conemu.github.io/en/BadIssue.html
+  - We at Cmder only understand the English language.
```

Comment:
That sounds a bit rude :D "Please keep ticket language to English" or so?

Reply:
That sounded a bit unprofessional! I changed it.